skorch/callbacks/logging.py

```diff
@@ -728,3 +728,105 @@ def on_epoch_end(self, net, **kwargs):
     def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.writer.close()
+
+
+class SacredLogger(Callback):
+    """Logs results from history to Sacred.
+
+    Sacred is a tool to help you configure, organize, log and reproduce
+    experiments developed at IDSIA. See https://github.com/IDSIA/sacred.
```

Comment:
```suggestion
    experiments. Developed at IDSIA. See https://github.com/IDSIA/sacred.
```

Otherwise, it makes you think Sacred only works for "experiments developed at IDSIA" :)

Reply:
That is their official tagline, but I see how it might be confusing :smile: